src/olympia/activity/tests/test_views.py

```diff
@@ -244,9 +244,9 @@ def test_queries(self):
             'fiiiine', amo.LOG.REVIEWER_REPLY_VERSION, self.days_ago(0)
         )
         self._login_developer()
-        with self.assertNumQueries(17):
+        with self.assertNumQueries(18):
             # - 2 savepoints because of tests
-            # - 2 user and groups
+            # - 3 user and groups
```

Comment:
Why ?

Reply:
mmm yeah. It scales with the number of items returned, so that would be bad and should be avoided.